src/App.js

```diff
@@ -1,19 +1,22 @@
-import { BrowserRouter, Route, Routes } from 'react-router-dom'
-import './App.css'
-import Nav from './Components/Nav'
-import Bookpage from './Components/Bookpage'
-import Bookshelf from './Components/Bookshelf'
-import NotFound from './Components/NotFound'
+import { BrowserRouter, Route, Routes } from "react-router-dom"
+import "./App.css"
+import Nav from "./Components/Nav"
+import Bookshelf from "./Components/Bookshelf"
+import NotFound from "./Components/NotFound"
 
 function App() {
     return (
-        <div className='App'>
+        <div className="App">
             <BrowserRouter>
                 <Nav />
                 <Routes>
-                    <Route path='/' element={<Bookshelf />} />
-                    <Route path='/book/:id' element={<Bookpage />} />
-                    <Route path='*' element={<NotFound />} />
+                    <Route path="/" element={<Bookshelf claimed={0} />} />
+                    <Route
+                        path="/claimed"
+                        element={<Bookshelf claimed={1} />}
+                    />
```

Comment:
I don't like how one of these routes is on one line, the other is split. Make it consistent plz